### ISSUE TITLE
External link title in the transcription

### ISSUE THREAD

**Issue by cyberbaloo** [numerique-gouv/meet#1086]
## Bug Report

**Problematic behavior**
The transcription offers the possibility to download the document through an external link. The download link label is not explicit and the screen reader does not convey the fact that it is a document that opens in a new window.

**Expected behavior/code**
The link label should be on the terms "Download your recording" and not on "following this link" and it must indicate the opening of a new window.

Add a `title="Download your recording - new window"` attribute

`<p class="bn-inline-content"><em>Télécharger votre enregistrement en </em><a target="_blank" rel="noopener noreferrer nofollow" href="https://visio.numerique.gouv.fr/recording/4dd3f422-e0fc-45b2-8ea3-4f4d4a65d30a"><em>suivant ce lien</em></a></p>`

**Additional context/Screenshots**

<img width="419" height="35" alt="Image" src="https://github.com/user-attachments/assets/e4778190-9059-4172-8080-5d58438b4a7f" />

**Comment by lebaudantoine** [comment]
Hi @cyberbaloo,
Most of this issue falls under the docs team, as we rely on their link element to create this link.
We don’t have the technical ability to enhance this part on our side.

I can enhance a bit the copywriting if you prefer.

**Comment by lebaudantoine** [comment]
the link's text would be "Télécharger votre enregistrement ici (lien externe)"
